src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/java/util/WcsResponseInterceptor.java

```diff
@@ -0,0 +1,33 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package ${package}.util;
+
+import de.terrestris.shogun2.util.interceptor.WcsResponseInterceptorInterface;
+import de.terrestris.shogun2.util.model.Response;
+
+/**
+ * This class demonstrates how to implement the WcsResponseInterceptorInterface.
+ * 
+ * @author Daniel Koch
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+public class WcsResponseInterceptor implements WcsResponseInterceptorInterface{
```

Comment:
Couldn't this trivial implementation be a basic/default implementation that is located in SHOGun, but not in the archetype?